src/user/row/style.js

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@material-ui/core'
+
+const paper = {
+  paddingTop: '1vh',
+  paddingBottom: '1vh',
+  textAlign: 'center',
+  fontSize: '1vw',
+}
+
+export const useStyle = makeStyles(() => ({ paper }))
```

Comment:
Same goes here. Let's rename this field from `paper` to `root` so that we can use the `classes` prop on the components that need this style.

Reply:
Done.